fastplotlib/layouts.py

```diff
@@ -111,9 +119,19 @@ def __init__(
                 renderer=renderer
             )
 
+            self.indexers[i, j] = GridPlotIndexer((i,j))
+
         self._animate_funcs: List[callable] = list()
         self._current_iter = None
 
+    def __getitem__(self, index: Union[tuple[int, int], str]):
+        if type(index) == str:
+            for indexer in self.indexers:
```

Comment:
Going off from the comment in the PR, iterate through `self.subplots` instead of the indexers.